packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts

```diff
@@ -1,12 +1,36 @@
-import { defineComponent } from 'vue'
-import ColorCSSVariablesUpdater from './ColorCSSVariablesUpdater.demo.vue'
+import { VaChip, VaButton } from '../../components'
+import { useColors } from '../../composables'
 
 export default {
-  title: 'ColorCSSVariablesUpdater',
-  component: ColorCSSVariablesUpdater,
+  title: 'services/ColorCSSVariablesUpdater',
 }
 
-export const Default = defineComponent({
-  components: { ColorCSSVariablesUpdater },
-  template: '<ColorCSSVariablesUpdater/>',
+export const Default = () => ({
+  components: { VaChip, VaButton },
+  setup () {
+    const colors = useColors()
+    const change = (color) => {
+      colors.setColors({ danger: color })
+    }
+
+    return {
+      change,
+    }
+  },
+  template: `
+    <div class="mb-6">
+      <a style="color: var(--va-danger)">
+        This is title danger color
+      </a>
+      <va-chip color="danger">This is vue component</va-chip>
+    </div>
+
+    Change danger color to:
+    <va-button @click="change('#4ab2e3')" color="#4ab2e3">
+      danger = blue
+    </va-button>
+    <va-button @click="change('#e34b4a')" color="#e34b4a">
+      danger = red
```

Comment:
We were repeating danger = in text above and button text

```suggestion
      red (#e34b4a)
```